cli/Cargo.toml

```diff
@@ -101,4 +101,4 @@ nix = "0.20.0"
 OriginalFilename = "deno.exe"
 LegalCopyright = "© Deno contributors & Deno Land Inc. MIT licensed."
 ProductName = "Deno"
-FileDescription = "A secure runtime for JavaScript and TypeScript."
+FileDescription = "Deno: A secure runtime for JavaScript and TypeScript"
```

Comment:
Removed period because nothing else in my task manager has a period.

Also, used a colon because it's similar to what node.js does: 
![image](https://user-images.githubusercontent.com/1609021/113353004-0f90c000-930b-11eb-93eb-190ac40e8171.png)